docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-meraki-source.md

```diff
@@ -0,0 +1,114 @@
+
+If the Source has any issues during any one of these states, it is placed in an **Error** state.
+
+When you delete the Source, it is placed in a **Stopping** state. When it has successfully stopped, it is deleted from your Hosted Collector. On the Collection page, the Health and Status for Sources is displayed. Use Health Events to investigate issues with collection. You can click the text in the Health column, such as **Error**, to open the issue in Health Events to investigate.
+
+Hover your mouse over the status icon to view a tooltip with details on the detected issue.
+
+## Setup and Configuration
+
+In this configuration, you will set up an Meraki source account and configure it to be authorized and authenticated to use device logs and alerts from MERAKI API.
+To obtain an Meraki auth token, follow the steps below:
+1. Log in to the [Meraki](https://dashboard.meraki.com/) application.
+1. Navigate to the **Organization**, under **Configure** section, select **Settings** page.<br/> <img src={useBaseUrl('img/send-data/cisco-meraki-org-settings.png')} alt="cisco-meraki-org-settings.png" width="500" />
+1. Check the **Enable access to the Cisco Meraki Dashboard API** <br/> <img src={useBaseUrl('img/send-data/cisco-meraki-org-enable-api.png')} alt="cisco-meraki-org-enable-api.png" width="800" />
+1. Click **Save Changes** at the bottom of the page.<br/> <img src={useBaseUrl('img/send-data/cisco-meraki-org-save.png')} alt="cisco-meraki-org-save.png" width="400" />
+1. Navigate to your profile by clicking your name in the upper right corner and select **My profile**.<br/> <img src={useBaseUrl('img/send-data/cisco-meraki-my-profile.png')} alt="cisco-meraki-my-profile.png" width="500" />
+1. Scroll down to **API access** and click **Generate new API key**.<br/> <img src={useBaseUrl('img/send-data/cisco-meraki-generate-key.png')} alt="cisco-meraki-generate-key.png" width="400" />
+1. Save your new API key into your preferred password vault for later use.<br/> <img src={useBaseUrl('img/send-data/cisco-meraki-save-key.png')} alt="cisco-meraki-save-key.png" width="500" />
+
+For more detailed steps with troubleshooting examples, visit the official documentation [here](https://developer.cisco.com/meraki/api-v1/#!authorization/authorization).
+
+## Create a Cisco Meraki Source
+
+When you create an Cisco Meraki Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+
+To configure Cisco Meraki Source:
+1. In Sumo Logic, select **Manage Data** > **Collection** > **Collection**.
+1. On the Collectors page, click **Add Source** next to a **HostedCollector**.
+1. Select Cisco Meraki.<br/> <img src={useBaseUrl('img/send-data/cisco-meraki-sumo-source-select.png')} alt="cisco-meraki-sumo-source-select.png" width="150" />
```

Comment:
I've swapped this out with a cleaner, hi-res image for you. And sized it down to look good.

```suggestion
1. Select Cisco Meraki.<br/> <img src={useBaseUrl('img/send-data/cisco-meraki-sumo-source-select.png')} alt="cisco-meraki-sumo-source-select.png" width="80" />
```